WEBROOOOT/portfolio.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	
+	<link rel="stylesheet" href="style.css">
+
+	 <title>P0RTFOLIO</title>
+	</head>
+<nav> 
+
+		<a href="index.html">(It's All) ALL ABOUT ME</a> 
+	 	<a href="portfolio.html">PORTF0LIO</a> 
+
+
+	 </nav>
+
+</center>
+
+</head>
+
+<body>
+
+<h1> <img src="77eyes.gif"></h1>
+<br> 
+<br>
+<br>
+<h2><img src="LILSPARKLYTREE.gif"></h2>
+
+ <h3> ///SOME RECENT PR0JECTS/// </h3>
+
+ 	<style> 
```

Comment:
The ```<style>``` tag should really just be including inside the ```<head>``` tag.  Also, perhaps this style could be moved into your css file?